index.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/utility.css">
+    <link rel="stylesheet" href="css/main.css">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+
+    <title>SIB</title>
+</head>
+<body>
+    <header class="header">
+        <div class="container header__wrapper">
+            <img class="logo" src="D:\FrontDevelop\front-template-main\pictures\logo.png" alt="IKIT Logotype">
+
+            <nav class="header__nav nav">
+                <ul class="nav__menu">
+                    <li class="nav__item"><a href="#" class="nav__link active">Главная</a></li>
```

Comment:
`<a href="#" class="nav__link active">Главная</a>`
 тут должен быть модификатор nav__link--active
```suggestion
                    <li class="nav__item"><a href="#" class="nav__link nav__link--active">Главная</a></li>
```